src/components/LogView/LogLineText.js

```diff
@@ -51,17 +99,41 @@ export default class LogLineText extends React.PureComponent<Props> {
       (this.props.endRange < 0 || this.props.lineNumber <= this.props.endRange)) {
       searchWords = this.props.highlightText;
     }
-    return (
-      <span ref={this.setRef} onDoubleClick={this.props.handleDoubleClick}>
-        <Highlighter
-          highlightClassName={'findResult' + this.props.lineNumber}
-          caseSensitive={this.props.caseSensitive}
-          unhighlightStyle={style}
-          highlightStyle={highlightStyle}
-          textToHighlight={this.props.text}
-          searchWords={searchWords}
-        />
-      </span>
-    );
+
+    if (this.props.prettyPrint && this.prettyPrintedText && this.prettyPrintedText.length > 1) {
```

Comment:
Should this be >0?

Reply:
No- my function will return a single string containing the full text of the line if it doesn't contain any JSON, the array will contain one item in that case.